packages/core/src/util/sequelize.ts

```diff
@@ -13,7 +13,7 @@ export const dbCredsSchema = z.object({
 /**
  * This function is used to initialize sequelize for lambda functions.
  */
-export function initSequelizeForLambda(dbCreds: string) {
+export function initSequelizeForLambda(dbCreds: string, logging = true) {
```

Comment:
Sure you want `logging` defaulting to `true`?

Reply:
oh, no, this was used for testing purposes.. ill take it out

Reply:
AFAIK, `logging`, if not specified, is set to true. 
I'm adding a method where I didn't want the queries to be logged, while also not affecting the existing functions that call this function. That's why I set it to true. Should I set it to false and find the existing callers and pass `true` to it instead?